porcupine/default_keybindings.tcl

```diff
@@ -3,11 +3,13 @@
 # Use Command on mac, Control on other systems
 if {[tk windowingsystem] == "aqua"} {
     set contmand Command
+    set alt_ish Control
     event add "<<RightClick>>" <Button-2>
     event add "<<RightClick>>" <Control-Button-1>
     event add "<<WheelClick>>" <Button-3>
 } else {
     set contmand Control
```

Comment:
Can you also rename `contmand` to `control_ish`?

Reply:
ok!